Client/Frontend/Browser/HomePanel/BraveShieldStatsView.swift

```diff
@@ -144,7 +144,7 @@ class StatView: UIView {
     
     fileprivate var titleLabel: UILabel = {
         let label = UILabel()
```

Comment:
We could also possibly change this to use `.then` instead. But this for me is optional.

Reply:
Yeah, this is some tech debt (e.g. this is not new / edited). I'm not going to adjust this now, since I want to fully rebuild this entire view "soon", lol.